style.css

```diff
@@ -0,0 +1,187 @@
+* {
+	box-sizing: border-box;
+}
+body{
+	margin: 0;
+	padding: 0;
+}
+nav a{
+	text-decoration: none;
+	font-family: 'Didact Gothic', sans-serif;
+	color: #F0F9F9;
+	font-weight: bold;
+	text-transform: uppercase;
+	padding: 20px;
+	font-size: 22px;
+}
+.menu-right{
+	float: right;
+}
+.nav{
+	position: fixed;
+	top: 0;
+	width: 100%;
+	padding: 20px;
+	overflow: hidden;
+}
+.open{
+	background-image: url("https://images.pexels.com/photos/956981/milky-way-starry-sky-night-sky-star-956981.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260");
+	background-repeat: no-repeat;
+	background-size: cover;
+	min-height: 100vh;
+}
+h1{
+	font-family: 'Didact Gothic', sans-serif;
+	font-weight: bold;
+	color: #F0F9F9;
+	margin: 0;
+	padding-bottom: 10px;
+}
+.socials{
+	padding: 20px;
+	position: absolute;
+	bottom: 5vh;
+	padding-left: 80px;
+}
+img, .socials{
+	display: inline-block;
+	margin-right: 10px;
+}
+.logos{
+	width: 60px;
+	height: 60px;
+}
+main{
+	background-color: #0F1221;
+	color: #F0F9F9;
+	padding-left: 80px;
+	padding-right: 80px;
+}
+h2{
+	font-family: 'Didact Gothic', sans-serif;
+	font-weight: bold;
+	padding: 20px;
+	padding-left: 0;
```

Comment:
Try not to mix a general padding rule with specific ones. This could be written as `padding: 20px 0;`